instance/static/html/instance/index.html

```diff
@@ -10,7 +10,7 @@
   <!-- Instances List -->
   <div class="large-3 columns">
     <ul class="side-nav side-instance-list">
-      <li ng-repeat="instance in instanceList | filter: {is_shut_down: false}"
+      <li ng-repeat="instance in instanceList | filter: {is_archived: false}"
```

Comment:
I think we should filter on the server side.  Archived instance aren't used in any way on the client side, so there is no point in querying and transferring them.